.ddev/commands/web/install-openmage

```diff
@@ -0,0 +1,137 @@
+SAMPLE_DATA_KEEP_FLAG=''
+USE_DEFAULT_FLAG=''
+
+while getopts 'dskq' flag; do
+  case "${flag}" in
+    d) USE_DEFAULT_FLAG='true' ;;
+    s) SAMPLE_DATA_FLAG='true' ;;
+    k) SAMPLE_DATA_KEEP_FLAG='true' ;;
+    q) QUIET_INSTALL_FLAG='true' ;;
+  esac
+done
+
+
+#quiet install
+if [[ "${QUIET_INSTALL_FLAG}" ]]; then
+  USE_DEFAULT_FLAG='true'
+  SAMPLE_DATA_FLAG='true'
+fi
+
+LOCALXML="${ROOT}/app/etc/local.xml"
+if [ -f "${LOCALXML}" ]; then
+  if [[ "${QUIET_INSTALL_FLAG}" ]]; then
+    DELETE='y'
+  else
+    read -r -p "OpenMage is already installed. Delete local.xml and drop database[y/N]:" DELETE
+    DELETE=${DELETE,,} # to lower
+  fi
+
+  if [[ "${DELETE}" =~ ^(yes|y) ]]; then
+    mysql -u db -h db -e "DROP SCHEMA IF EXISTS db; CREATE SCHEMA db;";
+    rm "${LOCALXML}"
+  else
+    exit 1
+  fi
+fi
+
+# install sample data
+if [[ "${SAMPLE_DATA_FLAG}" ]]; then
+  INSTALL_SAMPLE_DATA='y'
+else
+  read -r -p "Install sample data? [y/N]" INSTALL_SAMPLE_DATA
+  INSTALL_SAMPLE_DATA=${INSTALL_SAMPLE_DATA,,} # to lower
+fi
+
+if [[ $INSTALL_SAMPLE_DATA =~ ^(yes|y) ]]; then
+  SAMPLE_DATA_URL=https://github.com/Vinai/compressed-magento-sample-data/raw/master/compressed-magento-sample-data-1.9.2.4.tgz
+  SAMPLE_DATA_DIRECTORY="${ROOT}/.sampleData"
+  SAMPLE_DATA_FILE=sample_data.tgz
+
+  if [[ ! -d "${SAMPLE_DATA_DIRECTORY}" ]]; then
+    echo "Creating backup directory"
+    mkdir -p "${SAMPLE_DATA_DIRECTORY}"
+  fi
+
+  cd "${SAMPLE_DATA_DIRECTORY}" || exit
+  if [[ ! -f "${SAMPLE_DATA_DIRECTORY}/${SAMPLE_DATA_FILE}" ]]; then
+    echo "Downloading sample data"
+    wget "${SAMPLE_DATA_URL}" -O "${SAMPLE_DATA_FILE}"
+  fi
+
+  echo "Uncompress sample data"
+  tar xf "${SAMPLE_DATA_FILE}"
+
+  echo "Copy sample data"
+  cp -r magento-sample-data-1.9.2.4/* "${PWD}/"
+
+  echo "Import sample data"
+  mysql -u db -h db db < "${SAMPLE_DATA_DIRECTORY}/magento-sample-data-1.9.2.4/magento_sample_data_for_1.9.2.4.sql"
+
+  # remove sample data
+  if [[ "${SAMPLE_DATA_KEEP_FLAG}" ]]; then
+    rm -rf $(
+      find . -maxdepth 1 -type f -name "*" ! -name "${SAMPLE_DATA_FILE}")
+  else
+    cd "${ROOT}" || exit
+    rm -rf "${SAMPLE_DATA_DIRECTORY}"
+  fi
+fi
+
+cd "${ROOT}" || exit
+
+if [[ "${USE_DEFAULT_FLAG}" ]]; then
+  ADMIN_USER='admin'
+  ADMIN_FIRSTNAME='OpenMage'
+  ADMIN_LASTNAME='User'
+  ADMIN_EMAIL='admin@example.com'
+  ADMIN_PASSWORD='veryl0ngpassw0rd'
+  DB_PREFIX=''
+else
+  read -r -p "Admin user [admin]:" ADMIN_USER
+  ADMIN_USER=${ADMIN_USER:-admin}
+  read -r -p "Admin firstname [OpenMage]:" ADMIN_FIRSTNAME
+  ADMIN_FIRSTNAME=${ADMIN_FIRSTNAME:-OpenMage}
+  read -r -p "Admin lastname [User]:" ADMIN_LASTNAME
+  ADMIN_LASTNAME=${ADMIN_LASTNAME:-User}
+  read -r -p "Admin email [admin@example.com]:" ADMIN_EMAIL
+  ADMIN_EMAIL=${ADMIN_EMAIL:-admin@example.com}
+  read -r -p "Admin password [veryl0ngpassw0rd]:" ADMIN_PASSWORD
+  ADMIN_PASSWORD=${ADMIN_PASSWORD:-veryl0ngpassw0rd}
+  read -r -p "Database prefix []:" DB_PREFIX
```

Comment:
This should be "Tables prefix" not "Database prefix". Internal it is db_prefix which is correct but the meaning is completely different. When you install OM in browser the input label is "Tables Prefix".

![tables_prefix](https://github.com/OpenMage/magento-lts/assets/8360474/bfebca08-390b-4b9a-868f-13dc40806a95)